mongodb/Dockerfile

```diff
@@ -1,10 +1,15 @@
 # Base image
 FROM mongo:5.0.5
 
-# Create keyfile
-RUN openssl rand -base64 756 > /mongo-keyfile
+# Enviornment variable
```

Comment:
@ChristianMcc @ttbarnes I have updated `MongoDB` dockerfile to ensure Docker build process can be customised as per `ENABLE_REPLICA_SET` environment variable more information on the PR description. A quick review from you both will be great.